src/iop/toneequal.c

```diff
@@ -2423,9 +2423,10 @@ void gui_focus(struct dt_iop_module_t *self, gboolean in)
   if(!in)
   {
     //lost focus - stop showing mask
+    const gboolean was_mask = g->mask_display;
     g->mask_display = FALSE;
     gtk_toggle_button_set_active(GTK_TOGGLE_BUTTON(g->show_luminance_mask), FALSE);
-    dt_dev_reprocess_center(self->dev);
+    if(was_mask) dt_dev_reprocess_center(self->dev);
```

Comment:
merged a bit fast, I think this can be simplified as discussed in the other similar PR.

Reply:
@jenshannoschwalm : You may want to fix that the same way it was done on the others two module.

Reply:
I think this is already OK.

Reply:
Indeed :( I read too fast or I read the wrong patch as it is clearly not what I remember having seen :)